### DISCUSSION TITLE
CS-PMP%40Wisc/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#173] [Announcements]
# CS-PMP%40Wisc/

https://opencs.app/CS-PMP%40Wisc/

**Comment by yuhengcai1** [comment]
建议重新评估难度，我拿了UIUC的MCS然后被PMP拒了，而且我这种情况不算少数，群友拿了UIUC的MSCS都被PMP拒了

**Reply by godjoem** [reply]
只能说录取标准不一样我，我被你说的这几个拒了被Wisc录了

**Reply by HanH9** [reply]
感觉应该分到哪个难度呢
